Makefile

```diff
@@ -117,8 +117,14 @@ privileged-test: check-skopeo $(TESTDATA)
 	go test -failfast -tags needprivileges,$(BUILD_LABELS),containers_image_openpgp -v -trimpath -race -timeout 15m -cover -coverpkg ./... -coverprofile=coverage-dev-needprivileges.txt -covermode=atomic ./pkg/storage/... ./pkg/cli/... -run ^TestElevatedPrivileges
 
 $(TESTDATA): check-skopeo
-	$(shell mkdir -p ${TESTDATA}; cd ${TESTDATA}; mkdir -p noidentity; ../scripts/gen_certs.sh; cd ${TESTDATA}/noidentity; ../../scripts/gen_nameless_certs.sh; cd ${TOP_LEVEL}; skopeo --insecure-policy copy -q docker://public.ecr.aws/t0x7q1g8/centos:7 oci:${TESTDATA}/zot-test:0.0.1;skopeo --insecure-policy copy -q docker://public.ecr.aws/t0x7q1g8/centos:8 oci:${TESTDATA}/zot-cve-test:0.0.1)
-	$(shell chmod -R a=rwx ${TESTDATA})
+	mkdir -p ${TESTDATA}; \
+	cd ${TESTDATA}; ../scripts/gen_certs.sh; \
+	mkdir -p noidentity; cd ${TESTDATA}/noidentity; ../../scripts/gen_nameless_certs.sh; \
+	cd ${TOP_LEVEL}; \
+	skopeo --insecure-policy copy -q docker://public.ecr.aws/t0x7q1g8/centos:7 oci:${TESTDATA}/zot-test:0.0.1; \
+	skopeo --insecure-policy copy -q docker://public.ecr.aws/t0x7q1g8/centos:8 oci:${TESTDATA}/zot-cve-test:0.0.1; \
+	skopeo --insecure-policy copy -q -f oci docker://jetty:9.4.51-jre8-alpine-eclipse-temurin oci:${TESTDATA}/zot-cve-java-test:0.0.1; \
```

Comment:
@rchincha we need an image containing jar files uploaded to `docker://public.ecr.aws/t0x7q1g8`